source/_integrations/tedee.markdown

```
@@ -48,10 +48,73 @@ Local Access Token:
+  - action: lock.lock
+    metadata: {}
+    data: {}
+    target:
+      entity_id: lock.lock_a1b2
+```
+
+{% endraw %} {% enddetails %}
+
+## Kown Limitations
+
+This integration only supports functionality that is available locally. This means that 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)